torch/onnx/utils.py

```diff
@@ -470,7 +471,10 @@ def forward(self, x):
             * ``True``: export all ``nn.Module`` forward calls as local function nodes.
             * Set of type of nn.Module: export ``nn.Module`` forward calls as local function nodes,
               only if the type of the ``nn.Module`` is found in the set.
-
+        full_model_check (bool, default False): if True, ONNX checker will give the exported model a
+            strict shape type inference check to make sure it's a valid ONNX model. Notice that only a
+            valid ONNX model can be executed on ONNXRUNTIME. Default to False, as users might run the
```

Comment:
I wonder how it plays with `operator_export_type` that are not `ONNX`?

Reply:
It won't check non-ONNX according to:
https://github.com/pytorch/pytorch/blob/6865a35c08d9e1ad7fd4daf481fa2803a8655f00/torch/onnx/utils.py#L1543-L1549